### PR TITLE
Avoid escaped new lines in version.js generator

### PR DESCRIPTION
**Pull request by johnjbarton**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -270,7 +270,8 @@ src/codegeneration/ParseTreeTransformer.js: \
 	node $^ > $@
 
 src/runtime/version.js: package.json
-	echo "// generated in Makefile.\nexport let version = '$(PACKAGE_VERSION)';\n" > $@
+	echo "// generated in Makefile." > $@
```
Comment (arv) [inline Makefile]: Please do

```
@echo ...
```

No need to have make output the command line here

**Comment by arv** [comment]
LGTM

**Comment by arv** [comment]
Is this still needed?

**Comment by johnjbarton** [comment]
no

**Pull request closed without merging**